app/policies/podcast_policy.rb

```diff
@@ -20,4 +24,12 @@ def account_id
   def account_id_was
     resource.account_id_was
   end
+
+  class Scope < Scope
+    def resolve
+      # TODO: this is hacky
+      uris = token.authorized_account_ids(:read_private).map { |id| "/api/v1/accounts/#{id}" }
```

Comment:
I feel like someday (after deprecating CMS)... we should go through and change all these API-account-strings to just integers.